src/importkoaladialog.cpp

```diff
@@ -213,7 +213,7 @@ bool ImportKoalaDialog::tryChangeKey(int x, int y, char* key, quint8 mask, int h
 
         for (int i=0; i<totalMasks; ++i)
         {
-            if (mask && (1<<i))
+            if (mask && (1<<i) != 0)
```

Comment:
thanks. looks good to me

Reply:
I think that is incorrect .. probably should be: if (mask & (1<<i))

Reply:
as said in the other issue, if (mask && (1<<i) != 0) doesnt make sense either. (1<<i) is always != 0 (except when i becomes too large - then the result would be undefined behaviour).